### PR TITLE
Updating ingress-controller, default-backend and fixing permission for ClusterRole

### PR DESCRIPTION
**Pull request by BradErz**
Upgrading nginx-ingress-controller to 0.9.0-beta.10 and default backend server to 1.3.

Fixing the ClusterRole to allow permission to create/update config map as it complains. 

Fixes: #2895

### CODE REVIEW

**Comment by chrislovecnm** [comment]
@aledbf do you mind reviewing?

**Comment by aledbf** [comment]
@chrislovecnm sure

File: addons/ingress-nginx/v1.6.0.yaml [hunk 1]
```diff
@@ -199,12 +199,7 @@ rules:
   - ""
   resources:
   - events
-  verbs:
-  - create
-- apiGroups:
-  - ""
-  resources:
-  - endpoints
+  - configmaps
```
Comment (aledbf) [inline addons/ingress-nginx/v1.6.0.yaml]: This permissions are too open. Please check https://github.com/kubernetes/ingress/tree/master/examples/rbac/nginx
We only need to update one configmap used to store the leader election.

**Comment by BradErz** [comment]
@aledbf Heya, sorry for getting back to you so late. Had a busy week!

I refactored it to match the nginx rbac recommendations, now it's using a role and cluster role. I tested on my test cluster and seems just fine.

**Comment by justinsb** [comment]
Closing & reopening to trigger CI

**Pull request merged**